python/ray/serve/deployment.py

```diff
@@ -561,7 +561,7 @@ def deployment_to_schema(
         "num_replicas": None
         if d._deployment_config.autoscaling_config
         else d.num_replicas,
-        "max_concurrent_queries": d.max_concurrent_queries,
+        "max_concurrent_queries": d.max_ongoing_requests,
```

Comment:
Hmm, maybe I'm missing some context on why we need to forward those `max_concurrent_queries` to `max_ongoing_requests`. In the event when a user passes both, wouldn't that results deployment setting with the `max_ongoing_requests` value in both places and unable to retrieve the original if needed?? I feel internally when we "get" the value, we should make proper defaults and fallbacks between the two. But when we "set" them, they should be one-to-one else it'll be very confusion which one is which

Reply:
> unable to retrieve the original if needed
Hmm, the behavior here is that if `max_ongoing_requests` is set, we will ignore `max_concurrent_queries`; do we still need to be able to retrieve `max_concurrent_queries` if it's being overridden?

Reply:
I believe so. But if we don't need the ability of getting `max_concurrent_queries`, then we can probably just drop the field altogether (just not set it at all).

Reply:
The confusing part is setting them to something they are not

Reply:
Talked offline, there is currently no way for us to retrieve back `max_concurrent_queries ` and the chances for user needing to debug the override behavior is pretty low. We would just leave this as is